src/pages/ParticipantList.js

```diff
@@ -84,7 +85,9 @@ export default function ParticipantList({ setUser }) {
             <br />
             {participantList}
             <br />
-            <GreenButton onClick={createParticipant}><ButtonText>Add random participant</ButtonText></GreenButton>
+            <GreenButton onClick={createParticipant}>
+                <ButtonText>Add random participant</ButtonText>
+            </GreenButton>
```

Comment:
Why not put ButtonText inside the GreenButton?
Do you need ButtonText somewhere that is not inside a button?